docs/DevelopmentSetup/docker_installation_example.rst

```diff
@@ -1,101 +1,89 @@
 
-* Return to top level folder with ``cd ..``
+-  Return to top level folder with ``cd ..``
 
 .. figure:: ../images/installation_docker/step1_docker.png
-  :width: 600
-  :align: center
-  :alt: Building the FLINT.example image using Docker
+   :alt: Building the FLINT.example image using Docker
+   :align: center
+   :width: 600px
 
-  Building the FLINT.example image using Docker
+   Building the FLINT.example image using Docker
+
+Commands to run using docker - stock result written to screen and
+results files create (./Run_Env/*.csv): :
 
-Commands to run using docker - stock result written to screen and results files create (./Run_Env/*.csv):
 ::
 
-  # from repository root folder
+   # from repository root folder
 
-  # For Linux
-  docker run --rm -v $(pwd)/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_example.json --config config/libs.base.simple.json --logging_config logging.debug_on.conf"
+   # For Linux
+   docker run --rm -v $(pwd)/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_example.json --config config/libs.base.simple.json --logging_config logging.debug_on.conf"
 
-  # For Windows
-  docker run --rm -v %cd%/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_example.json --config config/libs.base.simple.json --logging_config logging.debug_on.conf"
+   # For Windows
+   docker run --rm -v %cd%/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_example.json --config config/libs.base.simple.json --logging_config logging.debug_on.conf"
 
 For the RothC example, you may run this command:-
 
 ::
 
-    # For Linux
-    docker run --rm -v $(pwd)/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_rothc_example.json --config config/libs.base_rothc.simple.json --logging_config logging.debug_on.conf"
+   # For Linux
+   docker run --rm -v $(pwd)/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_rothc_example.json --config config/libs.base_rothc.simple.json --logging_config logging.debug_on.conf"
 
-    # For Windows
-    docker run --rm -v %cd%/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_rothc_example.json --config config/libs.base_rothc.simple.json --logging_config logging.debug_on.conf"
+   # For Windows
+   docker run --rm -v %cd%/Run_Env:/usr/local/run_env -ti moja/flint.example:bionic bash -c "cd /usr/local/run_env/; moja.cli --config config/point_rothc_example.json --config config/libs.base_rothc.simple.json --logging_config logging.debug_on.conf"
 
 .. figure:: ../images/installation_docker/step2_docker.png
-  :width: 600
```

Comment:
Screenshot not visible!
https://moja-global-documentation--73.org.readthedocs.build/en/73/DevelopmentSetup/windows_installation.html#running-the-project